platform/ui/tailwind.config.js

```diff
@@ -435,9 +436,10 @@ module.exports = {
     },
     padding: (theme) => theme('spacing'),
     placeholderColor: (theme) => theme('colors'),
-    stroke: {
+    stroke: (theme) => ({
+      ...theme('colors'),
```

Comment:
cc @galelis here's the setting that will allow us to use `stroke-primary-main` to colorize SVG strokes. With that, we will be able to set different colors for fill and stroke. :)